lib/kino/bridge.ex

```diff
@@ -176,6 +176,14 @@ defmodule Kino.Bridge do
     end
   end
 
+  @doc """
+  Returns information about the running app.
+  """
+  @spec get_app_info() :: {:ok, map()} | {:error, request_error()}
+  def get_app_info() do
+    with {:ok, reply} <- io_request(:livebook_get_app_info), do: reply
```

Comment:
Should we return `%{type: :none}` if the `io_request` fails instead? This way we don't need to ask users to unpack the OK tuple.

Reply:
`Kino.Bridge` is lower level, we have the fallback in `Kino.Hub.app_info`.